netbox/core/api/views.py

```diff
@@ -95,7 +95,7 @@ class ObjectTypeViewSet(ReadOnlyModelViewSet):
     filterset_class = filtersets.ObjectTypeFilterSet
 
 
-class BaseRQViewSet(viewsets.ViewSet):
+class BaseRQViewSet(viewsets.GenericViewSet):
```

Comment:
I'm not sure it makes sense to employ GenericViewSet here as it adds methods such as `get_queryset()` which aren't applicable, as the view doesn't use a model. Could we instead just add declare a `get_serializer_class()` method?